clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java

```diff
@@ -270,6 +271,17 @@ private boolean isNodeFree(int nodeId) {
         return !nodesWithPendingRequests.contains(nodeId);
     }
 
+    public void setCallbackHandlerConfig(boolean isAcknowledgeCommitCallbackHandlerConfigured) {
```

Comment:
So, I'd call this `setAcknowledgementCommitCallbackRegistered` and propagate consistent naming throughout.